userguide/advanceduse/reverse-tethering.rst

```diff
@@ -1,34 +1,76 @@
 Reverse tethering
 =================
 
-This tutorial helps you to connect your Ubuntu Touch device to your computer to access the Internet on it.
+This tutorial helps you to connect your Ubuntu Touch device to your computer to access the internet on it.
 This is useful if there is no available Wi-Fi connection or you don't have a data subscription on your Ubuntu Touch device.
 
-Prerequisite: A Ubuntu Touch device with "Developer mode" on, attached via USB to the computer connected to the Internet.
+Prerequisite: A Ubuntu Touch device with "Developer mode" on, attached via USB to the computer connected to the internet.
+
+Tested: 2023-01-02 with Ubuntu Touch 16.04 (OTA-24) and Ubuntu 22.04.
 
 Steps
 -----
 
-1. On the Ubuntu Touch device: ``android-gadget-service enable rndis``
+1. On the Ubuntu Touch device:
+
+  - Run: ``android-gadget-service enable rndis``
```

Comment:
This get's rendered in a strange way. Looks like a quote. (Note: you can build the docs locally, or check how they come out when rendered in the cloud: https://docs.ubports.com/en/latest/contribute/documentation.html#final-check-of-your-contribution ) 

I think this has to do with the indentation. Alas, I notice that this is actually already broken in the current version. So, well, let me know if you feel up to check/fix this, otherwise never mind.

Reply:
Ok, I got it. The `-` of the bullet list must be directly under the first letter of the surrounding list. See: [restructuredtext](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists)